src/main/java/king/Storage.java

```diff
@@ -120,17 +120,19 @@ public boolean persistTaskList(TaskList taskList) {
      * and returns the corresponding tasks found in
      * a taskList.
      *
-     * @param keyword keyword to search for.
+     * @param keywords keyword to search for.
      * @return TaskList containing the tasks with the keywords
```

Comment:
Despite the sophisticated function of this class, you have made your code readable and clean.